### ISSUE TITLE
Alt text on collections book covers

### ISSUE THREAD

**Issue by LeighKP** [pressbooks/pressbooks-book-directory-fe#316]
Request:
I would like to be able to add alt text to the book covers in the "Featured Collections" space to have alt text.

Why:
I _think_ this might be an accessibility issue. This space should have a descriptor of the book cover in order to attribute those books as well.
